api/api.go

```diff
@@ -91,6 +92,10 @@ const (
 	SpecSharedv4ServiceName  = "sharedv4_svc_name"
 	SpecFastpath             = "fastpath"
 	SpecAutoFstrim           = "auto_fstrim"
+	SpecBackendType          = "backend"
+	SpecBackendPureBlock     = "pure_block"
+	SpecBackendPureFile      = "pure_file"
+	SpecPureFileExportRules  = "pure_export_rules"
```

Comment:
We could make this `SpecNFSExportRules` so that other vendors that support NFS can use it?

Reply:
change requested by @adityadani . I think. it makes sense, right now it's for FB only, and so far we don't have a standard for it yet,